plugins/shared_filesystem_storage/app/javascript/widgets/app/components/castellum/configuration/edit.jsx

```diff
@@ -190,6 +146,17 @@ const FormBody = ({ close, errors }) => {
           labelClass="control-label secondary-label"
         >
           <Form.Input elementType="input" type="number" min="0" />
+          <div className="checkbox">
+            <label>
+              <Form.Input
+                elementType="input"
+                type="checkbox"
+                name="minimum_free_is_critical"
+                disabled={values.free_minimum == ""}
+              />
+              <span>Treat crossings of this constraint as a critical usage level</span>
```

Comment:
To me it's not immediately clear what this text means but I'm also not deeply into the topic. Will our users understand what this means?

Reply:
This is the best phrasing that I can come up with that ought to be clear while fitting in a single line. The word "critical" refers to a few lines above in the same form, where users configure low/high/critical thresholds.

Reply:
I suggest: `If usage crosses this threshold, mark it as a critical level.`

Reply:
I don't see how this is better.

Reply:
I now rephrased it to Stefans suggestion: `When crossing this threshold, resize without delay` which should be more clear now.